src/rp/util/logic.clj

```diff
@@ -0,0 +1,7 @@
+(ns rp.util.logic)
+
+(defn xor
+  [p q]
```

Comment:
The 3-input XOR truth table shows that (xor T T T) is true. http://mathworld.wolfram.com/XOR.html